### PR TITLE
Add errors connected with pasting images using pastetools

### PR DESCRIPTION
**Pull request by Comandeer**
I've added short descriptions of the new errors connected with pasting images.

Closes ckeditor/ckeditor4#4334.

### CODE REVIEW

**Review by f1ames — APPROVED** [review]
LGTM :+1:

**Comment by f1ames** [comment]
@godai78 could you take a look?

File: docs/guide/dev/deep_dive/errors/README.md [hunk 1]
```diff
@@ -134,6 +134,22 @@ This article contains the list of CKEditor 4 error codes and their explanation.
...
+* Location: `plugins/pastetools/filter/image.js`
+* Description: Images couldn't be correctly extracted from the RTF content provided by [Paste from Word](https://ckeditor.com/cke4/addon/pastefromword) or [Paste from LibreOffice](https://ckeditor.com/cke4/addon/pastefromlibreoffice) plugins.
+* Additional data:
+	* `rtf`: Amount of images extracted from RTF.
+	* `html`: Amount of images present in HTML.
+
+## pastetools-unsupported-image
+
+* Location: `plugins/pastetools/filter/image.js`
+* Description: One of the images extracted from the RTF content provided by [Paste from Word](https://ckeditor.com/cke4/addon/pastefromword) or [Paste from LibreOffice](https://ckeditor.com/cke4/addon/pastefromlibreoffice) plugins is in unsupported format.
```
Comment (godai78) [inline docs/guide/dev/deep_dive/errors/README.md:148]:
```suggestion
* Description: One of the images extracted from the RTF content provided by the [Paste from Word](https://ckeditor.com/cke4/addon/pastefromword) or [Paste from LibreOffice](https://ckeditor.com/cke4/addon/pastefromlibreoffice) plugins is in unsupported format.
```

File: docs/guide/dev/deep_dive/errors/README.md [hunk 2]
```diff
@@ -134,6 +134,22 @@ This article contains the list of CKEditor 4 error codes and their explanation.
 * Additional data:
 	* `path`: A path to the vendor library.
 
+## pastetools-failed-image-extraction
+
+* Location: `plugins/pastetools/filter/image.js`
+* Description: Images couldn't be correctly extracted from the RTF content provided by [Paste from Word](https://ckeditor.com/cke4/addon/pastefromword) or [Paste from LibreOffice](https://ckeditor.com/cke4/addon/pastefromlibreoffice) plugins.
```
Comment (godai78) [inline docs/guide/dev/deep_dive/errors/README.md:140]:
```suggestion
* Description: Images couldn't be correctly extracted from the RTF content provided by the [Paste from Word](https://ckeditor.com/cke4/addon/pastefromword) or [Paste from LibreOffice](https://ckeditor.com/cke4/addon/pastefromlibreoffice) plugins.
```

**Review by godai78 — APPROVED** [review]
Minor: articles.

**Pull request merged**